examples/webpack.config.js

```diff
@@ -42,26 +43,28 @@ module.exports = async (env) => {
 
     // see also production proxy at /examples/server/src/endpoints/proxy.js
     const proxy = {
-        '/gdc': {
-            target: backendUrl,
-            secure: false,
-            cookieDomainRewrite: '',
-            onProxyReq: (proxyReq) => {
-                console.log('proxy', '/gdc', proxyReq.path); // eslint-disable-line no-console
-                if (proxyReq.method === 'DELETE' && !proxyReq.getHeader('content-length')) {
-                    // Only set content-length to zero if not already specified
-                    proxyReq.setHeader('content-length', '0');
-                }
-
-                proxyReq.setHeader('host', backendUrl.split('/')[2]); // White labeled resources are based on host header
-                proxyReq.setHeader('referer', backendUrl);
-                proxyReq.setHeader('origin', null);
+        "/gdc": {
+            "changeOrigin": true,
+            "cookieDomainRewrite": "localhost",
+            "secure": false,
+            "target": backendUrl,
+            "headers": {
+                "host": backendUrl,
+                "origin": null
+            },
+            onProxyReq(proxyReq) {
+                proxyReq.setHeader('accept-encoding', 'identity')
             }
         },
         '/api': {
             target: 'http://localhost:3009',
             secure: false,
             onProxyReq: (req) => {
+                console.log('proxy', '/gdc', req.path); // eslint-disable-line no-console
```

Comment:
Now we have two console logs on each call. Is this necessary?

Reply:
I just moved this.

Reply:
I think it's unnecessary too. But i did not want to change look and feel :D